### ISSUE TITLE
[BUG] Quickstart is broken

### ISSUE THREAD

**Issue by jeandut** [owkin/fedeca#6]
**Describe the bug**
Running the quickstat produces an assertion error on the fact that the length of strategies and metrics_dicts_list are different

**To Reproduce**
```python
import pandas as pd
from fedeca.utils.survival_utils import CoxData
from fedeca import FedECA

# Let's generate 1000 data samples with 10 covariates
data = CoxData(seed=42, n_samples=1000, ndim=10)
df = data.generate_dataframe()

# We remove the true propensity score
df = df.drop(columns=["propensity_scores"], axis=1)


fed_iptw = FedECA(ndim=10, treated_col="treatment", event_col="event", duration_col="time", robust=True)
fed_iptw.fit(df, n_clients=4, split_method="split_control_over_centers", split_method_kwargs={"treatment_info": "treatment"}, data_path="./data", robust=True, backend_type="simu")

fed_iptw = FedECA(ndim=10, treated_col="treatment", event_col="event", duration_col="time", robust=True)
fed_iptw.fit(df, n_clients=4, split_method="split_control_over_centers", split_method_kwargs={"treatment_info": "treatment"}, data_path="./data", robust=True, backend_type="simu")
```

**Expected behavior**
The code should run smoothly.

**Desktop (please complete the following information):**
- OS: MacOS
- Python version 3.9
- Code version [e.g. 0.02 or commit hash]

**Comment by jeandut** [comment]
As it turns out we fell into the classical trap of mutable default args in Python. I'll change that.

**Comment by jeandut** [comment]
Solved by #7
